icons/sfml.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>SFML icon</title><path d="M15.937 2.191c2.082 1.558 4.25 3.006 6.25 4.668 1.161.962 2.127 2.434 1.717 4.007-.56 2.286-1.395 4.495-2.088 6.743-.637 1.853-1.078 3.918-2.571 5.29-1.458 1.093-3.375.733-5.077.804-2.467-.039-4.942.1-7.4-.128-1.396-.125-2.845-.806-3.388-2.188-.885-2.18-1.503-4.46-2.258-6.688C.552 12.824-.3 10.893.106 8.902c.538-1.744 2.3-2.584 3.636-3.647C5.76 3.831 7.682 2.258 9.804.992c1.202-.724 2.774-1.027 4.021-.23.747.41 1.423.932 2.112 1.43zm.175 2.257l-.836 3.221c1.233.922 2.511 1.786 3.71 2.75a452.127 452.127 0 01-2.593 8.049c-.405.471-1.304.15-1.938.239-2.274-.018-4.551.037-6.822-.033-.904-2.67-1.756-5.36-2.614-8.045.028-.613.933-.87 1.382-1.303.757-.547 1.51-1.102 2.267-1.65l-.836-3.221C6.148 5.705 4.41 6.89 2.77 8.197c-1.15 1.057-1.038 2.79-.465 4.112.844 2.544 1.627 5.11 2.497 7.643.555 1.4 2.135 1.984 3.542 1.857 2.762-.023 5.525.025 8.285-.026 1.496-.1 2.543-1.423 2.862-2.806.824-2.646 1.733-5.267 2.532-7.92.374-1.46-.563-2.867-1.774-3.598-1.387-.993-2.757-2.008-4.137-3.01zm-4.294-2.277c-.865.096-2.003.575-2.396 1.28l2.088 8.07c.358-.08 1.019.233 1.01-.325l2.043-7.876c-.78-.64-1.688-1.235-2.745-1.149z"/></svg>
```

Comment:
Thanks for your work on this @MiguelMJ 🎉
I've just compared your PR (red) to the source (black) and it appears that during the optimization process some of the corners were smoothed out. Could you take another look at this - and use a precision of 4 on [SVGOMG](https://jakearchibald.github.io/svgomg/) if necessary?

![image](https://user-images.githubusercontent.com/1830867/103998532-05d84000-5194-11eb-8f32-218363c1c49b.png)

Reply:
Done! I think the problem was caused because I used the Simplify option of Inkscape... Now I didn't. I had to manually resolve a collinear segment, but now it should be ready 😅

Reply:
Thanks @MiguelMJ - though it appears one side of the inner shape is shorter than the other now!
![image](https://user-images.githubusercontent.com/1830867/104239159-fa19a180-5451-11eb-9805-6b3a469790c3.png)

Other than that, the icon is spot on!

Reply:
Oops! Sorry. Can you tell me what software do you use to compare them? So I can avoid submitting incorrect icons again. Thanks!

Reply:
Sure thing! So I generally use Figma for comparing SVGs, but you can overlay them in any vector editing software.

Reply:
You can easily do it as well in Inkscape. 😊  Just use File --> Import, select the other SVG, give it the same size, center it and then give the one on the top a different color and an opacity.

Reply:
How didn't I think of that :man_facepalming:
Thanks a lot!

Reply:
Hey @MiguelMJ - have you been able to look at this change at all?

Reply:
I'm afraid I've been really busy, so no 😞

Reply:
That's fine - there's no rush! Just wanted to make sure you still wanted to work on this 😆

Reply:
Of course 💪🏼  and thanks 😄